scipy/interpolate/interpolate.py

```diff
@@ -2315,6 +2315,10 @@ class RegularGridInterpolator(object):
     avoids expensive triangulation of the input data by taking advantage of the
     regular grid structure.
 
+    If any of `points` are one-dimensional, linear intepolation will return an
```

Comment:
Should talk of "dimension of size 1", not about 1-dimensional which is a different thing.